spec/features/service_entries/update_service_entry_spec.rb

```diff
@@ -90,8 +90,11 @@
 
           # Remove and add back the second option
           select "lorem_223 | ipsum", from: 'tag_guids_fromList'
+          page.find('body').native.send_keys :escape
+
           find('button[title=remove]').click
           select "lorem_223 | ipsum", from: 'tag_guids_toList'
+          #page.find('body').native.send_keys :escape
```

Comment:
if this isn't needed we should remove it.

Reply:
:this:

Reply:
agree